frontend/package.json

```diff
@@ -15,8 +15,8 @@
     "test": "pnpm -r test"
   },
   "devDependencies": {
-    "prettier": "^3.0.3",
-    "npm-run-all": "^4.1.5"
+    "npm-run-all": "^4.1.5",
```

Comment:
Is flipping the order of these needed?

Reply:
i haven't made this change, maybe it got flipped after linting? 

@seankmartin